### PR TITLE
N18: A/B Street

### PR DESCRIPTION
**Pull request by dabreegster**
_Part of #455_

### CODE REVIEW

File: content/posts/newsletter-018/index.md [hunk 1]
```diff
@@ -149,6 +149,26 @@ bringing more interesting crafting scenarios into the game.
 [Antorum]: https://ratwizard.dev/dev-log/antorum
 [@dooskington]: https://twitter.com/dooskington
 
+# [A/B Street]
```
Comment (ozkriff) [inline content/posts/newsletter-018/index.md:152]:
```suggestion
## [A/B Street]
```

**Comment by ozkriff** [comment]
Could you please avoid including the coordination issue's number to the commit message in the future? It creates some unnecessary noise in the issue because of how GitHub's UI works:
image

File: content/posts/newsletter-018/index.md (hunk 1 above)
Comment (dabreegster) [reply content/posts/newsletter-018/index.md]: Whoops, thanks! I need to install mdlint or remember to check the CI

**Comment by dabreegster** [comment]
> Could you please avoid including the coordination issue's number to the commit message in the future?

Yep, sorry about that! Blindly followed the example from recent commits.

**Review by ozkriff — APPROVED** [review]
:+1:

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**